addons/l10n_pk/demo/account_demo.py

```diff
@@ -0,0 +1,324 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models, Command
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = "account.chart.template"
+
+    @api.model
+    def _get_demo_data(self, company=False):
+        if company.account_fiscal_country_id.code == "PK":
+            if company.state_id:
+                return {
+                    'res.partner': self._get_demo_data_partner(company),
+                    'account.move': self._get_demo_data_move(company),
+                    'res.config.settings': self._get_demo_data_config_settings(company),
+                }
+            else:
+                _logger.error(
+                    'Error while loading Accounting(Pakistan) demo data in the company "%s".',
+                    company.name
+                )
+        else:
+            return super()._get_demo_data(company)
+
+    @api.model
+    def _get_demo_data_partner(self, company=False):
```

Comment:
Same goes for this method.

@jco-odoo method i.e. `_get_demo_data_move` are defined in `account` which is inheritable by different modules.
But methods i.e. `_get_demo_data_partner`, `_get_demo_data_config_settings` is only defined in `l10n_in`.

What do you suggest for that ? Do we make inheritable method in `account`? or consider renaming the method i.e. `_get_demo_data_partner_in`/`_get_demo_data_partner_pk`? or else we risk the overriding the methods while installing both localisation together.

Reply:
I have already prefixed these methods with the module name 😶